src/utils/parser/constants.mjs

```diff
@@ -273,71 +297,20 @@ export const DOC_TYPES_MAPPING_NODE_MODULES = {
 
   'brotli options': 'zlib.html#class-brotlioptions',
 
-  'cluster.Worker': 'cluster.html#class-worker',
```

Comment:
I assume this is what the `// Transform Node.js types like 'vm.Something'.` is doing?

Reply:
Yep